### ISSUE TITLE
request validated data, training and output files

### ISSUE THREAD

**Issue by sandymartin** [Dowell-Lab/FStitch#16]
In order to help troubleshoot our installation which is not working it would be helpful if you could post a bedGraph file, a training file and the corresponding successful output files for FStitch train and segment.

**Comment by magruca** [comment]
@sandymartin I have added additional, more explicit, error message outputs. If you git pull, make clean, and re-compile, hopefully these will help you further identify where things are going wrong. I mentioned this in previous communication, but there is a functional Docker/Singularity container, as well. You can also now install FStitch via conda:
conda install -c dowell-lab fstitch

**Comment by sandymartin** [comment]
Thanks for letting me know, Margaret.
________________________________

**Comment by magruca** [comment]
Of course! One other thing to note -- we will soon have all of our FStitch data (including training files, segment data, and bidirectional annotations) that you are requesting here integrated into a new/updated version of our publish nascent database @ nascent.colorado.edu which will include all published nascent data. We're hoping to have the new release launched by October.
